### ISSUE TITLE
How to access published date in component?

### ISSUE THREAD

**Issue by enyo** [storyblok/storyblok-svelte#281]
I want to display the `published_at` date in my blog post. But ` <StoryblokComponent />` gets `blok={data.story.content}` and the `published_at` property is inside `data.story` so is not available to my page component.

What is the recommended way to get the date down to my page component?

**Comment by sebasvdonk** [comment]
Cant you just give the StoryblokComponent the parent property.
```svelte
<StoryblokComponent blok={data.story} />
```

**Comment by enyo** [comment]
I don’t think so, because the `StoryblokComponent` expects an actual `blok` as the `blok` property, and looks at the `blok.component` field to see which component to instantiate.

I solved it like this:

```svelte
  <StoryblokComponent
    blok={data.story.content}
    {...{
      publishedAt: new Date(data.story.first_published_at),
    }}
  />
```

And inside my `Blogpost.svelte`:

```svelte
<script lang="ts">
  export let blok: any
  export let publishedAt: Date
</script>
```
